tests/integration/defs/kimi_k3_sa_harness.py

```
@@ -0,0 +1,534 @@
+            llm = _build_llm(ckpt, tp, "off", adp)
+            baseline = _generate(llm, prompt_texts, max_tokens, want_logprobs)
+            llm.shutdown()
+
+    llm = _build_llm(ckpt, tp, spec_mode, adp)
+    completions = _generate(llm, prompt_texts, max_tokens, want_logprobs)
+    llm.shutdown()
+    if output_json:
+        _dump_completions(output_json, completions, want_logprobs)
+
+    failures = []
+    for comp, (prompt, expected) in zip(completions, prompt_set):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)